vgen/src/main/resources/io/gapi/vgen/client/common.snip

```diff
@@ -0,0 +1,32 @@
+# TODO: Consolidate with other language-independent common snippet files.
```

Comment:
Agreed that consolidation is worthwhile, but let's address this in a follow-up PR.